packages/js/admin-e2e-tests/src/fixtures/plugins.ts

```diff
@@ -47,15 +47,18 @@ async function deactivateAndDeletePlugin( pluginName: string ) {
 }
 export async function deactivateAndDeleteAllPlugins( except: string[] = [] ) {
 	let plugins = await getPlugins();
+	let skippedPlugins = [];
 	const promises = [];
 	for ( const plugin of plugins ) {
 		const splitPluginName = plugin.plugin.split( '/' );
 		const slug = splitPluginName[ 1 ] || splitPluginName[ 0 ];
 		if ( ! except.includes( slug ) ) {
 			promises.push( deactivateAndDeletePlugin( plugin.plugin ) );
+		} else {
+			skippedPlugins.push( slug );
```

Comment:
Not all the same plugins passed into `except` are installed, so make sure we only check the plugins length of the actual installed skipped plugins.